Project SF/Views/Tabs/Profile/ProfileView.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ProfileView.swift
+//  Project SF
+//
+//  Created by Roman Esin on 13.07.2020.
+//
+
+import SwiftUI
+
+struct ProfileView: View {
+    var body: some View {
+        NavigationView {
+            Text("Profile View")
```

Comment:
Where is the profile name, username, pfp, etc?

Reply:
Otherwise its fine

Reply:
This PR is only for creating new folder. The profile data is still to be added

Reply:
So you'll add it later?

Reply:
Yeah

Reply:
K